scripts/kovan-deployment.ts

```diff
@@ -0,0 +1,52 @@
+
+const hre = require("hardhat");
+const dotenv = require("dotenv");
+
+async function main() {
+
+  let recipient = process.env.RECIPIENT
+  const contracts = {
+    AAVE_POOL: process.env.AAVE_POOL,
+    ADAI: process.env.ADAI,
+    DAI: process.env.DAI
+  }
+  const [deployer] = await hre.ethers.getSigners();
+  console.log(
+    "Deploying contracts with the account:",
+    deployer.address
+  );
+
+  const Token = await hre.ethers.getContractFactory("BazrToken");
+  const btokenDepoymentTx = await Token.deploy();
+
+  console.log("Account balance:", (await deployer.getBalance()).toString());
+  let btokenBlueprint = await btokenDepoymentTx.deployed();
+  const Vault = await hre.ethers.getContractFactory("Vault");
+  const vaultDepoymentTx = await Vault.deploy();
+
+  let vaultBlueprint = await vaultDepoymentTx.deployed();
+
+  const VaultFactory = await hre.ethers.getContractFactory("VaultFactory");
```

Comment:
just a heads up, but with AccessControl and permissions implemented, the deployer will have admin rights to call `createVault` and `createBToken` so we might have to specify the signer here

`VaultFactory = await ethers.getContractFactory("VaultFactory", { signer: owner });` (see PR  #9)

once we deploy our "final" version this deployer will have the sole rights to call `createVault` and `createBToken` or `grantRole` to other addresses that should have that access

Reply:
ack 👍